database/mysql.go

```diff
@@ -92,6 +104,11 @@ func (my *mysql) Connect(ctx context.Context) (*sql.DB, error) {
 				my.connections.With("state", "idle").Set(float64(stats.Idle))
 				my.connections.With("state", "inuse").Set(float64(stats.InUse))
 				my.connections.With("state", "open").Set(float64(stats.OpenConnections))
+				my.counters.With("stat", "wait_count").Set(float64(stats.WaitCount))
```

Comment:
This is called `counter` in the metric definition.

Reply:
Yup, just caught that myself too.